evm/src/main/java/org/hyperledger/besu/evm/gascalculator/CancunGasCalculator.java

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright contributors to Hyperledger Besu.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.evm.gascalculator;
+
+import org.hyperledger.besu.datatypes.DataGas;
+
+/** The Cancun gas calculator. */
+public class CancunGasCalculator extends LondonGasCalculator {
+
+  public static final int CANCUN_DATA_GAS_PER_BLOB = 131072; // 2^17
```

Comment:
maybe we can link the EIP in a comment for future dev

Reply:
done in the class doc